linters/djlint/djlint.test.ts

```diff
@@ -0,0 +1,4 @@
+import { linterCheckTest } from "tests";
+
+// TODO(Tyler): We will eventually need to add a couple more test cases involving failure modes and other packages.
+linterCheckTest({ linterName: "djlint" });
```

Comment:
Can you also add a `linterFmtTest` and rerun the test to generate a snapshot? Sorry I should've caught this in the first pass. This should mirror sqlfluff test, but without the `preCheck` setup.